mne/preprocessing/ica.py

```diff
@@ -1370,6 +1371,63 @@ def plot_components(self, picks=None, ch_type=None, res=64, layout=None,
                                    image_interp=image_interp,
                                    head_pos=head_pos)
 
+    def plot_properties(self, inst, picks=None, axes=None, dB=True,
+                        cmap=None, plot_std=True, topo_args=None,
+                        image_args=None, psd_args=None, figsize=None,
+                        show=True):
+        """Display component properties: topography, epochs image, ERP,
+        power spectrum and epoch variance.
+
+        Parameters
+        ----------
+        inst: instance of Epochs or Raw
+            The data to use in plotting properties.
+        picks : int | array-like of int | None.
+            The components to be displayed. If None, plot will show the first
+            five sources in the order as fitted. If more than one components
+            were chosen in the picks - each one will be plotted in a separate
+            figure. Defaults to None.
+        axes: list of matplotlib axes | None
+            List of five matplotlib axes to use in plotting: [topo_axis,
+            image_axis, erp_axis, spectrum_axis, variance_axis]. If None a new
+            figure with relevant axes is created. Defaults to None.
+        dB: bool
+            Whether to plot spectrum in dB. Defaults to True.
+        cmap: matplotlib colormap | None
+            Colormap to use in both topoplot and epochs image. If None topoplot
+            and epochs image use their default "RdBu_r" colormap.
+            Defaults to None.
+        plot_std: bool | float
+            Whether to plot standard deviation in ERP/ERF and spectrum plots.
+            Defaults to True, which plots one standard deviation above/below.
+            If set to float allows to control how many standard deviations are
+            plotted. For example 2.5 will plot 2.5 standard deviation
+            above/below.
+        topo_args : dict | None
+            Dictionary of arguments to plot_topomap. If None - doesn't pass any
+            additional arguments. Defaults to None.
+        image_args : dict | None
+            Dictionary of arguments to plot_epochs_image. If None - doesn't
+            pass any additional arguments. Defaults to None.
+        psd_args : dict | None
+            Dictionary of arguments to psd_multitaper. If None - doesn't pass
```

Comment:
...

Also func names get special formatting, I think at least double backticks. Check how it's done elsewhere.